var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -65,8 +65,9 @@ def cmake_args(self):
                 '-DALUMINUM_ENABLE_MPI_CUDA:BOOL=%s' % ('+ht' in spec))
 
         if '@:0.1,0.6.0:':
```

Comment:
This is missing an `in spec` or `spec.satisfies()`. Python strings always evaluate to True if non-empty. You can actually combine this line and the one below it.